Mathlib/SetTheory/ZFC/Ordinal.lean

```diff
@@ -114,6 +126,34 @@ theorem subset_of_mem (h : x.IsOrdinal) : y ∈ x → y ⊆ x :=
 theorem mem_trans (h : z.IsOrdinal) : x ∈ y → y ∈ z → x ∈ z :=
   h.isTransitive.mem_trans
 
+@[simp]
+protected theorem empty : IsOrdinal ∅ :=
+  ⟨.empty, fun _ _ H ↦ (notMem_empty _ H).elim⟩
+
+@[deprecated (since := "2025-07-30")] alias isOrdinal_empty := IsOrdinal.empty
+
+protected theorem sUnion (hx : ∀ y ∈ x, IsOrdinal y) : (⋃₀ x).IsOrdinal := by
+  refine ⟨.sUnion' fun y hy ↦ (hx y hy).isTransitive, @fun z a b hza hab hb ↦ ?_⟩
+  rw [mem_sUnion] at hb
+  obtain ⟨c, hcx, hbc⟩ := hb
+  exact (hx c hcx).mem_trans' hza hab hbc
+
+protected theorem sInter (hx : ∀ y ∈ x, IsOrdinal y) : (⋂₀ x).IsOrdinal := by
```

Comment:
Why is `IsOrdinal.mem_trans'` not defined as `mem_trans' {w : ZFSet} : w ∈ x → w.IsTransitive`? That should make these proofs a bit easier. And is `IsOrdinal.mem_trans` supposed to be an alias of `mem_trans'` with different implicitness? Currently it's a restatement of `IsOrdinal.isTransitive`.